lib/timeline/TimeStep.js

```diff
@@ -216,7 +216,17 @@ TimeStep.prototype.next = function() {
       case 'millisecond':  this.current.add(this.step, 'millisecond'); break;
       case 'second':       this.current.add(this.step, 'second'); break;
       case 'minute':       this.current.add(this.step, 'minute'); break;
-      case 'hour':         this.current.add(this.step, 'hour'); break;
+      case 'hour':
+        this.current.add(this.moment.duration(this.step, 'hour'));
+
+        // correct for daylight saving
+        // FIXME: use this.current.add(moment.duration(this.step, 'hour'))
+        // see http://momentjs.com/docs/#special-considerations-for-months-and-years
+        if (this.current.hours() % this.step !== 0) {
```

Comment:
To be honest, I don't understand this part. Can you explain in a few words what you are doing here?